ansible/roles/infra-osp-template-generate/tasks/locate_template.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Everything else should use this going forward, combined with a specific set of tasks called by main.yml
+# to configure the template creation for the specific cloud provider
+- name: Locate template for OSP cloud provider
+  when: cloud_provider == 'osp' # TODO: Make this "not AWS" so everything else can be more generic use without cloudformation reference
```

Comment:
Again is this necessary? this is in an infra-osp role - it should always be 'osp', shouldn't it?